commit 044755e2faeeca13bb77b2e9e638a45e6e90a5fa
Author: Florian Weimer <fweimer@redhat.com>
Date:   Tue Aug 30 13:30:03 2022 +0200

    resolv: Fix building tst-resolv-invalid-cname for earlier C standards
    
    This fixes this compiler error:
    
    tst-resolv-invalid-cname.c: In function ‘test_mode_to_string’:
    tst-resolv-invalid-cname.c:164:10: error: label at end of compound statement
         case test_mode_num:
              ^~~~~~~~~~~~~
    
    Fixes commit 9caf782276ecea4bc86fc94fbb52779736f3106d
    ("resolv: Add new tst-resolv-invalid-cname").
    
    (cherry picked from commit d09aa4a17229bcaa2ec7642006b12612498582e7)

diff --git a/resolv/tst-resolv-invalid-cname.c b/resolv/tst-resolv-invalid-cname.c
index ae2d4419b1978c02..63dac90e02d6cbc7 100644
--- a/resolv/tst-resolv-invalid-cname.c
+++ b/resolv/tst-resolv-invalid-cname.c
@@ -162,7 +162,7 @@ test_mode_to_string (enum test_mode mode)
     case gai_canon:
       return "gai_canon";
     case test_mode_num:
-      /* Report error below.  */
+      break;                    /* Report error below.  */
     }
   FAIL_EXIT1 ("invalid test_mode: %d", mode);
 }
